src/components/Homepage/HomeExp/KivaMultiCategoryGrid.vue

```diff
@@ -1,35 +1,40 @@
 <template>
-	<div class="md:tw-flex md:tw-items-start">
-		<div class="tw-mr-4">
-			<p class="tw-text-h3 tw-font-medium tw-mb-2 tw-text-secondary">
-				I want to support
-			</p>
-			<loan-category-selector-home-exp
+	<div class="row">
+		<div class="small-12 large-3 medium-3">
```

Comment:
The `row`, `small-12`, `large-3`, and `medium-3` classes are all from Foundation, not Tailwind, which has different breakpoints.

You can use the KvGrid component and `col-span` classes to do the same thing in Tailwind:
```vue
<kv-grid class="tw-grid-cols-12">
    <div class="tw-col-span-12 md:tw-col-span-3">
        ...
    </div>
    <div class="tw-col-span-12 md:tw-col-span-9">
        ...
    </div>
</kv-grid>
```